apps/system/locales/accessibility.en-US.properties

```diff
@@ -336,3 +336,20 @@ accessibility-stateUncheckedAbbr = ( )
 accessibility-statePressedAbbr = (x)
 # LOCALIZATION NOTE: "( )" is a braille abbreviation for unpressed
 accessibility-stateUnpressedAbbr = ( )
+
+# Hint dialogues for actionable events
+accessibility-pushbutton-hint = Double tap to activate
+accessibility-checkbutton-hint = Double tap to activate
+accessibility-combobox-hint = Press and hold options to focus
+accessibility-key-hint = Double tap to activate
+accessibility-link-hint = Double tap to activate
+accessibility-menuitem-hint = Double tap to activate
+#accessibility-'check menu item' = Double tap to activate
+#accessibility-'radio menu item' = Double tap to activate
+accessibility-option-hint = Double tap to activate
+accessibility-radiobutton-hint = Double tap to activate
+accessibility-slider-hint = Swipe up or down to progress
+accessibility-spinbutton-hint = Swipe up or down to increase/decrease
+accessibility-pagetab-hint = Double tap to activate
+accessibility-entry-hint = Press and hold keys to begin typing
```

Comment:
Are you sure about this one?  On my phone this hint only comes up when the keyboard is showing and the entry is already active.